### DISCUSSION TITLE
Manually timed adlist update

### DISCUSSION THREAD

**Discussion by TheCeet** [AdguardTeam/AdGuardHome#4217] [Q&A]
Is it possible to choose when I want to update my adlists?

Example:
ISP's in our country have FUP (limited bandwith monthly).
At night, usage counts for 50%. So it would be nice to tell AdGuardHome to update the lists at 03:00AM.

Now my only way is to keep awake at night and manually trigger the update button with update interval 24h.

I searched in the .yaml file but can't find anything relevant.

**Comment by ainar-g** [comment]
It's currently not possible.  See feature request #3044.
